core/neat/mixins/_grid-push.scss

```diff
@@ -0,0 +1,35 @@
+@charset "UTF-8";
+/// Push or pull a Neat grid column.
+///
+/// @argument {number (unitless)} $push [1]
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The grid to be used to generate the column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include grid-push(3);
+///   }
+///
+/// @example css
+///   .element {
+///     margin-left: calc((100% - (260px)) * 0.25 + 80px);
+///   }
+
+@mixin grid-push($push: false, $grid: $neat-default-grid) {
```

Comment:
@tysongach I would be fine with that. `null` to me says "do nothing" where as the default state is more of a reset. The use case would be that if i had a column with `@include grid-push(2);` on mobile but I wanted the margin to be the default `$gutter` on desktop, I would need to actively *reset* the value. Perhaps `@include grid-push(0);` would be an option but I found false to be more tidy.